openapi3/response_issue224_test.go

```diff
@@ -457,5 +457,5 @@ func TestEmptyResponsesAreInvalid(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, doc.ExternalDocs.Description, "See AsyncAPI example")
 	err = doc.Validate(context.Background())
-	require.EqualError(t, err, `invalid paths: the responses object MUST contain at least one response code`)
+	require.Contains(t, err.Error(), `the responses object MUST contain at least one response code`)
```

Comment:
Please keep `EqualError`. 
> `invalid paths: invalid path /pet: invalid operation POST: the responses object MUST contain at least one response code`

Reply:
I would have except the sample spec used in this test actually has multiple operations that can fail. It's not deterministic either so I don't know which one will fail validation first.

Reply:
Ah yes I have a branch that's working towards deterministic validation. I'll try and finish it soon.

Reply:
Rebased and updated this test.